processing/src/main/java/org/apache/druid/query/filter/SelectorDimFilter.java

```diff
@@ -118,21 +119,23 @@ public String getDimension()
 
   @Nullable
   @JsonProperty
+  @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
IMO, it would be better to omit the `@JsonInclude` here. `value` is a required parameter for the selector filter, and `value: null` represents an "is null" filter. It's weird for the canonical JSON form of an "is null" filter to not have any `value` at all.